Heading/Heading.js

```diff
@@ -113,6 +113,17 @@ const HeadingDecorator = compose(
 	Skinnable
 );
 
+HeadingDecorator.PropTypes = /** @lends sandstone/Heading.HeadingDecorator.prototype */ {
+	/**
+	 * Determines what triggers the header content to start its animation.
+	 *
+	 * @type {('hover'|'render')}
+	 * @default 'render'
+	 * @public
+	 */
+	marqueeOn: PropTypes.oneOf(['hover', 'render'])
+};
+
```

Comment:
I think this part should be removed.

https://github.com/enactjs/sandstone/commit/9184982d1d146858b07aae75e417a6d59019f4db#diff-b90f5f2a021576b34363ec36140158eb1aa5f133c23d42f1d5e01597105c159b

Reply:
Hmm.. Interesting. This code lives through auto-merging by Git.

Reply:
I don't know what happened but this code was not removed not this merging but the prior sync up with develop branch.